src/Utilities/List.f90

```diff
@@ -20,6 +20,7 @@ module ListModule
     procedure, public :: Add
     procedure, public :: Clear
     procedure, public :: Count
+    procedure, public :: Contains
```

Comment:
contains is a fortran keyword. This should probably be something else to avoid confusion.